p2panda-rs/src/secret_group/mls/group.rs

```diff
@@ -206,15 +237,19 @@ mod tests {
         let key_pair = KeyPair::new();
         let provider = MlsProvider::new();
 
+        // Create MLS group with one member
         let member = MlsMember::new(&provider, &key_pair).unwrap();
         let group_id = GroupId::random(&provider);
         let key_package = member.key_package(&provider).unwrap();
-        let mut group = MlsGroup::new(&provider, group_id, key_package).unwrap();
+        let mut group = MlsGroup::new(&provider, group_id, key_package.clone()).unwrap();
+
+        // Group is active and contains the owner of the group as the only member
         assert!(group.is_active());
+        assert_eq!(group.members(), vec![&key_package]);
 
+        // Owner can not decrypt its own messages (for forward secrecy)
```

Comment:
Hmmm, I don't understand this comment, could you expand a little?

Reply:
openmls deletes the session key material from this client as soon as it's been used (for encryption), other clients will do the same right after they decrypted the message. So basically you can do it only once and have to store the plaintext somewhere.

This is part of forward secrecy and improves the security around an attacker getting the private key not being able to decrypt old messages as the session keys are gone.